src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve(`First promise was resolved`);
+  });
+
+  setTimeout(() => {
+    reject(new Error(`First promise was rejected`));
+  }, 3000);
+});
```

Comment:
The first promise will always resolve on a click event, and the rejection after 3 seconds will never occur if a click happens first. Consider whether this behavior is intended.